content/en/logs/guide/aws-account-level-logs.md

```diff
@@ -0,0 +1,266 @@
+---
+title: AWS Account-Level Log Subscriptions
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+---
+
+You can use account-level log subscriptions in your AWS environment to automatically forward all of your CloudWatch logs to Datadog. With this setup, you don't need to manually configure log forwarding when you have a new log source, or when AWS releases a new service. You can also define your own selection criteria or filter pattern for granular control over which logs are forwarded.
+
+## Create an account-level log subscription
+
+For the simplest setup, use [CloudFormation](#cloudformation-recommended) to create a Kinesis Firehose and associated resources in each of your selected regions.
+
+### CloudFormation (recommended)
+
+1. Copy the URL of the CloudFormation template:
+
+{{< code-block lang="bash" filename="" disable_copy="false" >}}
+https://datadog-cloudformation-template.s3.amazonaws.com/aws_account_level_logs/main.yaml
+{{< /code-block >}}
+
+2. Go to [CloudFormation][1] in the AWS console.
+3. Click **Create stack**.
+    - Select `With new resources (standard)`.
+4. Leave the options to **Choose an existing template** and **Amazon S3 URL** checked.
+5. In the **Amazon S3 URL** field, paste the URL of the CloudFormation template.
+6. Click **Next**.
+7. In the **Stack name** field, provide a descriptive name such as `datadog-account-level-logs-stack`.
+8. In the **ApiKey** field, paste a valid [Datadog API key][4] value.
+9. In the **Regions** field, enter a comma-separated list of regions to enable metric streaming.
```

Comment:
I would change `regions` to `AWS Regions` so it's more clear what regions we're looking for.
Also, `enable metric streaming` should be something like `enable account-level logs subscription"

Reply:
Thanks for reviewing and catching that, updated in [c7373f1](https://github.com/DataDog/documentation/pull/25398/commits/c7373f188698e361872e2921903a94d1665d026e)